systemd/systemd.go

```diff
@@ -205,6 +207,11 @@ func New(rootDir string, mode InstanceMode, rep reporter) Systemd {
 	return &systemd{rootDir: rootDir, mode: mode, reporter: rep}
 }
 
+// New returns a Systemd that runs with preseed enabled in SystemMode and uses the given rootDir
+func NewWithPreseedMode(rootDir string, rep reporter) Systemd {
```

Comment:
as I mentioned, we probably don't want the term preseed in this package,   this is basically doing most of systemd job without calling it. So maybe  NewEmulation(...) or NewEmulationMode() ? and then the comment should mention what I described instead of preseed mode